oak_functions_enclave_service/src/lib.rs

```diff
@@ -0,0 +1,224 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#![no_std]
+
+extern crate alloc;
+
+use alloc::{format, string::ToString, sync::Arc, vec::Vec};
+
+use oak_attestation::handler::EncryptionHandler;
+use oak_core::sync::OnceCell;
+use oak_crypto::encryptor::EncryptionKeyHandle;
+pub use oak_functions_service::proto;
+use oak_functions_service::{
```

Comment:
moved this code out from `oak_functions_service/src/lib.rs` with close to no changes, though the diff shows it as new